src/UiPath.Workflow.Runtime/RuntimeArgument.cs

```diff
@@ -78,6 +78,8 @@ internal static IComparer<RuntimeArgument> EvaluationOrderComparer
 
     public ArgumentDirection Direction { get; private set; }
 
+    public string DisplayName => _bindingProperty?.DisplayName ?? Name;
```

Comment:
this can be a breaking change. Being a public property on a public type, NewtonsoftJson serialization will serialize it and if the workflow is resumed on an older Robot it will fail. Mark it internal instead